{{ cookiecutter.repo_name }}/src/{{ cookiecutter.package_dir_name }}/__init__.py

```diff
@@ -1,4 +1,23 @@
-from ._version import get_versions
+#!/usr/bin/env python
+##############################################################################
+#
+# (c) 2024 The Trustees of Columbia University in the City of New York.
+# All rights reserved.
+#
+# File coded by: {{ cookiecutter.author_name }}
```

Comment:
@Tieqiong this will need to be modified a bit too  I think @Sparks29032 is working on it.

We don't want 2024 hard-coded and we will have new wording for hte file coded by.  Just a heads-up.